src/stores/types/asset-api.ts

```diff
@@ -65,9 +65,15 @@ export enum Gebaeude {
     Z = 'Z',
 }
 
-export enum Standort {
-    In = 'IN',
-    Nd = 'ND',
+export const Standort = {
```

Comment:
Line 38: 'Standort' refers to a value, but is being used as a type here. You could use typeof 'Standort'. After that you have to fix the dependent type errors.